training/model/Makefile

```diff
@@ -20,4 +20,29 @@ download: image
 .PHONY: download-model
 download-model:
 	mkdir -p ../build/models
-	podman run -e HF_TOKEN -v ../build/models:/download:z -t model-downloader huggingface-cli download --exclude "*.pt" --local-dir "/download/$(MODEL_REPO)" "$(MODEL_REPO)"
+	podman run \
+		-e HF_TOKEN \
```

Comment:
where's the HF_TOKEN set?

Reply:
nm, i see it in the .github/workflow

Reply:
but dont we need to call it as an env variable tho like: `-e HF_TOKEN=$$HF_TOKEN` or as a make value passed: `-e HF_TOKEN=$(HF_TOKEN)?